bridgetown-core/lib/site_template/package.json.erb

```diff
@@ -27,11 +27,12 @@
     <% if options["use-postcss"] %>
     "postcss": "^8.1.9",
     "postcss-flexbugs-fixes": "^4.1.0",
-    "postcss-loader": "^4.1.0",
+    "postcss-loader": "^5.0.0",
     "postcss-preset-env": "^6.7.0",
     <% else %>
-    "node-sass": "^4.13.1",
-    "sass-loader": "^8.0.2",    
+    "fibers": "",
```

Comment:
Can we take this out if we're not using fibers by default?

Reply:
Fibers is needed along with sass-loader